iron-menu-behavior.html

```diff
@@ -23,7 +23,8 @@
   Polymer.IronMenuBehaviorImpl = {
 
     properties: {
-
+      _searchText: '',
```

Comment:
since it's just a private, just `this._searchText` when you need it is enough (or somewhere in ready/attached/whatever if you need a default value)-- this looks like you're doing it just for documentation purposes, and we no longer do that.